Schema/Resolvers.js

```diff
@@ -3,24 +3,24 @@ const { channels, basket } = require("../data");
 const resolvers = {
   Query: {
     // Channel
-    getAllChannel() {
+    channels() {
       return channels;
     },
-    getSportsChannel() {
+    sportsChannels() {
```

Comment:
This should probably be a parameter on `channels`, but it's okay for now

Reply:
you mean filter by channel and add variables like 

```
getChannel($catergory: String!) {
     getChannel($catergory: category) {
           id
           name
           category
     }
} 
```

Reply:
Yeah that's the idea, but with the name `channels`, not `getChannel`.  Note plural and don't think of this as a function call, it's a query for a field so the naming relates to the field type not an imperative/RPC type action (i.e. not verb-y)